From 4ee0836ff3f3c5ee14965da250bc549abcbcc95c Mon Sep 17 00:00:00 2001
From: Richard Hughes <richard@hughsie.com>
Date: Wed, 21 Feb 2018 19:01:57 +0000
Subject: [PATCH] Use a CNAME to redirect to the correct CDN for metadata

The current CDN (~$100/month) is kindly sponsored by Amazon, but that won't
last forever. In the future we can switch to a 'dumb' provider like BunnyCDN
for 1/10th of the cost.

Use a CNAME we control to make switching CDN providers easy in the future.
---
 data/remotes.d/lvfs-testing.conf | 2 +-
 data/remotes.d/lvfs.conf         | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/data/remotes.d/lvfs-testing.conf b/data/remotes.d/lvfs-testing.conf
index 75da08f1..41d9ffff 100644
--- a/data/remotes.d/lvfs-testing.conf
+++ b/data/remotes.d/lvfs-testing.conf
@@ -4,7 +4,7 @@
 Enabled=false
 Title=Linux Vendor Firmware Service (testing)
 Keyring=gpg
-MetadataURI=https://s3.amazonaws.com/lvfsbucket/downloads/firmware-testing.xml.gz
+MetadataURI=https://cdn.fwupd.org/downloads/firmware-testing.xml.gz
 ReportURI=https://fwupd.org/lvfs/firmware/report
 Username=
 Password=
diff --git a/data/remotes.d/lvfs.conf b/data/remotes.d/lvfs.conf
index 51d0f55e..4e971be6 100644
--- a/data/remotes.d/lvfs.conf
+++ b/data/remotes.d/lvfs.conf
@@ -4,6 +4,6 @@
 Enabled=true
 Title=Linux Vendor Firmware Service
 Keyring=gpg
-MetadataURI=https://s3.amazonaws.com/lvfsbucket/downloads/firmware.xml.gz
+MetadataURI=https://cdn.fwupd.org/downloads/firmware.xml.gz
 ReportURI=https://fwupd.org/lvfs/firmware/report
 OrderBefore=fwupd
-- 
2.14.3

